tests/integration/src/setup.rs

```diff
@@ -23,17 +23,95 @@ pub use frame_support::{
     traits::{OnFinalize, OnIdle, OnInitialize},
     weights::Weight,
 };
-pub use sp_core::H160;
+pub use pallet_evm::AddressMapping;
+pub use sp_core::{H160, H256, U256};
 pub use sp_runtime::{AccountId32, MultiAddress};
 
+pub use astar_primitives::ethereum_checked::AccountMapping;
+
 #[cfg(feature = "shibuya")]
 pub use shibuya::*;
 #[cfg(feature = "shibuya")]
 mod shibuya {
+    use super::*;
     pub use shibuya_runtime::*;
 
     /// 1 SBY.
     pub const UNIT: Balance = SBY;
+
+    // TODO: once Account Unification is finished, remove `alith` and `alicia`,
+    // which are mocks of two way account/address mapping.
+
+    /// H160 address mapped from `ALICE`.
+    pub fn alith() -> H160 {
+        h160_from(ALICE)
+    }
+
+    /// `AccountId32` mapped from `alith()`.
+    pub fn alicia() -> AccountId32 {
+        account_id_from(alith())
+    }
+
+    /// Convert `H160` to `AccountId32`.
+    pub fn account_id_from(address: H160) -> AccountId32 {
+        <Runtime as pallet_evm::Config>::AddressMapping::into_account_id(address)
+    }
+
+    /// Convert `AccountId32` to `H160`.
+    pub fn h160_from(account_id: AccountId32) -> H160 {
+        <Runtime as pallet_ethereum_checked::Config>::AccountMapping::into_h160(account_id)
+    }
+
+    /// Deploy an EVM contract with code.
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

LGTM!



<!-- This is an auto-generated comment by OSS CodeRabbit -->